egs/wsj0-mix/DeepClustering/model.py

```diff
@@ -33,8 +35,31 @@ def make_model_and_optimizer(conf):
     enc = fb.Encoder(fb.STFTFB(**conf['filterbank']))
     masker = ChimeraPP(int(enc.filterbank.n_feats_out/2), 2,
                        embedding_dim=20, n_layers=2, hidden_size=600, \
-                       dropout=0, bidirectional=True)
+                       dropout=0.5, bidirectional=True, \
+                       log=conf['filterbank']['log'])
     model = Model(enc, masker)
```

Comment:
Even if the STFT is not inverted at train time, the goal with eventually be to go back to the time domain. You might as well attach the iSTFT to the model, it will make the rest easier IMO.